jwst/lib/pointing_summary.py

```diff
@@ -0,0 +1,146 @@
+"""Pointing Summary
+
+Review contents of a set of given models for pointing information.
+Compare the calculated V1 and REFPOINT pointing with the proposed
+TARGET pointing.
+
+Examples
+--------
+>>> from jwst.datamodels import ImageModel
+>>> im = ImageModel()
+>>> im.meta.target.ra       =  90.75541666666666
+>>> im.meta.target.dec      = -66.56055555555554
+>>> im.meta.pointing.ra_v1  =  91.08142004561715
+>>> im.meta.pointing.dec_v1 = -66.60547868904696
+>>> im.meta.wcsinfo.ra_ref  =  90.70377653291781
+>>> im.meta.wcsinfo.dec_ref = -66.59540223936895
+>>> calc_pointing_deltas(im)
+    Delta(target=<SkyCoord (ICRS): (ra, dec) in deg
+        (90.75541667, -66.56055556)>, v1=<SkyCoord (ICRS): (ra, dec) in deg
+        (91.08142005, -66.60547869)>, refpoint=<SkyCoord (ICRS): (ra, dec) in deg
+        (90.70377653, -66.59540224)>, delta_v1=<Angle 0.13712727 deg>, delta_refpoint=<Angle 0.04044315 deg>)
+
+>>> calc_deltas([im])
+    <Table length=1>
+      exposure                  target                ...    delta_refpoint
+                               deg,deg                ...
+       object                   object                ...       float64
+    ------------ ------------------------------------ ... -------------------
+    <ImageModel> 90.75541666666666,-66.56055555555554 ... 0.04044314761499765
+"""
+from collections import defaultdict, namedtuple
+import logging
+
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+import astropy.units as u
+
+import jwst.datamodels as dm
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+__all__ = ['Delta', 'calc_pointing_deltas', 'calc_deltas']
+
+# Basic delta structure
+Delta = namedtuple('Delta', 'target, v1, refpoint, delta_v1, delta_refpoint')
```

Comment:
This might be nice as a class with `delta_v1` and `delta_refpoint` as property methods.  Could add a `__repr__` to make logging convenient, too.

Reply:
(never mind, this isn't used outside of this class, is it)

Reply:
Good point, or actually maybe that data class stuff. But, until a real use-case comes up, the line-in-sand is here.